libavcodec/vvc/vvcdec.c

```diff
@@ -240,6 +275,19 @@ static void bs_tl_init(TabList *l, VVCFrameContext *fc)
         TL_ADD(horizontal_bs[i], bs_count);
         TL_ADD(vertical_bs[i],   bs_count);
     }
+}
+
+static void bs_nz_tl_init(TabList *l, VVCFrameContext *fc)
+{
+    const VVCPPS *pps   = fc->ps.pps;
+    const int bs_width  = pps ? (pps->width >>  2) + 1 : 0;
+    const int bs_height = pps ? (pps->height >> 2) + 1 : 0;
+    const int bs_count  = bs_width * bs_height;
+    const int changed   = fc->tab.sz.bs_width != bs_width ||
+        fc->tab.sz.bs_height != bs_height;
```

Comment:
Is it better to align it with fc->tab.sz.bs_width != bs_width ||?
Like
```C++
const int changed   = fc->tab.sz.bs_width != bs_width ||
                      fc->tab.sz.bs_height != bs_height;
```
or not a new line. There are also long lines in the codes above.
```C++
const int changed   = fc->tab.sz.bs_width != bs_width || fc->tab.sz.bs_height != bs_height;
```

Reply:
sure, will do

Reply:
I combined bs with tu. so this line is not needed